eng/tools/TypeSpecValidation/src/rules/file-structure.ts

```diff
@@ -0,0 +1,27 @@
+import { globby } from "globby";
+import { Rule } from "../rule.js";
+import { RuleResult } from "../rule-result.js";
+
+export class FileStructureRule implements Rule {
```

Comment:
```suggestion
export class FolderStructureRule implements Rule {
```

I think `FolderStructure` is a slightly better name than `FileStructure`.